airflow2/docker-compose.yml

```diff
@@ -39,7 +39,8 @@
 version: '3'
 x-airflow-common:
   &airflow-common
-  image: ${AIRFLOW_IMAGE_NAME:-apache/airflow:2.0.1-python3.8}
+  build:
+    context: .
```

Comment:
this looks about right.

Also perhaps it is time for a little refactoring, moving the elk related containers into a subfolder.

Reply:
I agree. Please see https://meaningfy.atlassian.net/browse/SC-68